give-fbpt.php

```diff
@@ -0,0 +1,55 @@
+<?php namespace GiveFBPT;
+
+use GiveFBPT\Addon\Activation;
+use GiveFBPT\Addon\Environment;
+use GiveFBPT\Domain\AddonServiceProvider;
```

Comment:
The `Domain` domain is supposed to be renamed during the build process. Please update the folder and namespaces to a name which fits. Probably `FacebookPixel`.

Reply:
Ah - I see! Taken care of. Ref: https://github.com/impress-org/givewp-facebook-pixel-tracking/pull/8/commits/71af050fa19835c2421bc19c87c383397631f999

Reply:
@JasonTheAdams is it important that the `Domain` be unique to GiveWP somehow? Like `GiveWPPixelTracking` instead of a `Domain` that is most likely used by another plugin?

Reply:
Good question! The answer is no, simply because we're not using the same root namespace as GiveWP core. Core uses `Give\` as the root namespace, whereas this plugin is using `GiveFBPT\`. Therefore all of the underlying namespaces won't collide with core.